### DISCUSSION TITLE
Text not fully visible in Deepin Device Manager side menu

### DISCUSSION THREAD

**Discussion by xoascf** [linuxdeepin/developer-center#6158] [Troubleshooting | 故障排查]
This mainly affects Spanish localization.

https://github.com/linuxdeepin/developer-center/assets/28744392/99f86b77-7ff4-414a-a9e0-e9c759a0538d

**Comment by BLumia** [comment]
cc @babyfengfjx

**Comment by xoascf** [comment]
_So it has gotten worse, and I'm wondering if this will ever be fixed..._

![image](https://github.com/user-attachments/assets/7334ae23-1f43-4168-b337-403731414bef)

**Reply by BLumia** [reply]
I've forwarded this to the related team, hopefully this will be fixed in later updates.
